src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_COST = 40;
+  const DISCOUNT_DAYS_BASIC = 3;
+  const DISCOUNT_VALUE_BASIC = 20;
+  const DISCOUNT_DAYS_EXTENDED = 7;
+  const DISCOUNT_VALUE_EXTENDED = 50;
```

Comment:
It is better to keep static constants outside functions so that they are not recreated.